resources/views/guest/mainPage.blade.php

```diff
@@ -0,0 +1,255 @@
+                            <input class="form-control w-100 p-4 m-0 d-inline-block mr-5 job-search-input-guest" type="text" name="search" id="searchbox"
+                        placeholder="Search jobs (e.g. Work from Home)">
+                        <a href="#" class="btn btn-primary search-button" name="">Search Jobs </a>
+                        </div>
+                       </div>
+                   </form>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <div class="container mt-5">
+        {{-- Latest Job   --}}
+        <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Latest Job</h1>
+       @if ($jobs->count() > 0)
+            <div class="row">
+                @foreach ($jobs as $index => $job)
+                    @if($index < 8)
+                        <div class="col-lg-3 col-sm-6 khaled">
+                            <div class="card latest-job my-2">
+                                <div class="card-body box-shad">
+                                    <h5 class="card-title"><a href="">{{$job->title}}</a></h5>
+                                    <span class="card-text">{{$job->company->name}}</span>
+                                    <span> - {{$job->city->name}}</span>
+                                    <p class="card-text"><small class="text-muted">{{\Illuminate\Support\Carbon::now()->diffForHumans(\Illuminate\Support\Carbon::parse($job->created_at))}} Ago</small></p>
+                                </div>
+                            </div>
+                        </div>
+                    @endif
+                @endforeach
+            <div class="col-sm-12">
+                <div class="float-right my-3">
+                    <a href="#" >See all new Jobs on Land A Job</a>
+                </div>
+            </div>
+        </div>
+       @else
+        <div class="container text-center">
+            <h1>There is no jobs Yet</h1>
+        </div>
+       @endif
+
+        <hr class="my-4 mx-5">
+        {{--  Jobs By Career Level  --}}
+        <h1 class="mb-3 mt-5 font-weight-bold text-secondary">Jobs By Career Level</h1>
+      @if ($careerlevels->count() > 0)
+        <div class="row ">
+            @foreach ($careerlevels as $index => $careerlevel)
+                @if($index < 6)
+                    <div class="col-lg-4 col-sm-12 my-3">
+                        <div class="box-shad">
+                            <a href="#">
+                                <img class="career-level" src="{{asset('img/guest/career/'.$career[$index])}}" alt="Not Found">
+                            </a>
+                            <h5 class="position-absolute text-light text-bottom p-2">{{$careerlevel->name}}</h5>
+                        </div>
+                    </div>
+                @endif
+            @endforeach
+        </div>
+      @else
+            <div class="container text-center">
+                <h1>There is no career level Yet</h1>
+            </div>
```

Comment:
If this is possible and there are no career levels.
Don't show this section only, don't tell the user.